frontend/controller/actions/chatroom.js

```diff
@@ -121,7 +129,17 @@ export default (sbp('sbp/selectors/register', {
             foreignKey: cekOpts.foreignKey,
             meta: cekOpts.meta,
             data: cekOpts.data
-          }
+          },
+          encryptedOutgoingDataWithRawKey(CEK, {
```

Comment:
Can you leave a TODO comment here to figure out how to avoid having the developer interact with `encryptedOutgoingDataWithRawKey`?